example/migrations/main.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// migrations command demostrate the functionality of
+// user data migration for authenticated Github user
+// and list all the user migrations.
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+func FetchAllUserMigrations() ([]*github.UserMigration, error) {
+	ctx := context.Background()
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: "b453839afb5b7871b9dabfe8184b314747f6523a"},
```

Comment:
Please don't add access tokens to the repo... and if this is a valid one, you will now want to go and delete it as it has been exposed to the world.

`&oauth2.Token{AccessToken: "b45...access-token-here...3a"},`

Reply:
Yes. I have already deleted this access token. Now removed from code too.